packages/flutter/lib/src/material/drop_down.dart

```diff
@@ -411,28 +412,31 @@ class DropdownButtonHideUnderline extends InheritedWidget {
 class DropdownButton<T> extends StatefulWidget {
```

Comment:
I will put it on my TODO list.

https://github.com/flutter/flutter/issues/6975